web-console/4.0-designs/storage/object-storage-dashboard/object-storage-dashboard.md

```diff
@@ -0,0 +1,91 @@
+# Object Storage Dashboard￼
+
+## Vision
+Our main goal was to have an object's storage dashboard supported by MCG (NooBaa) that provide the user with at-a-glance information used for analysis and decision making. 
+
+## Object Dashboards
+
+![Object Storage Dashboard](img/object-storage-dashboard.png)
+
+**The objects dashboard includes the following cards:**
+- Details
+- Buckets (Object buckets and Object bucket claims)
+- Resources providers
+- Health
+- Data consumption 
+- Data resiliency 
+- Usage breakdown
+- Data Reduction 
+
+## Details Card
+![Details Crad](img/details-card.png)
+
+**The details card shows:**
+- connected service name and offer external link to the NooBaa (MCG) system. 
+- Provider name
+- MCG installed version
+
+## Buckets Card 
+![Buckets Crad](img/buckets-card.png)
+
+MCG - MCG/NooBaa Buckets are data containers that behave similarly to AWS S3 buckets. Users can set policies that will determine which resources will be user (per bucket) to store data.
+We will show a breakdown between static buckets and buckets claim. Each bucket type will have a status indication when something is wrong with X buckets (shows the number of buckets that are currently in warning/ error status). The number link should lead to the problematic buckets list. 
+
+## Resources Providers Card
+![Resources Providers](img/resource-card.png)
+This card list the different storage resources the user have. It can be a cloud based resource and a bare metal resource. For each type (AWS, google, ceph rgw…) we will show the number of resources and their status indication if something is wrong.
+**Only used resources types will be shown, no need to show types that are not in used. 
+
+## Health Card
```

Comment:
@yuvalgalanti Please add the following texts to the PR:

**Info icon texts:**
**Resources Providers:**
A list of all MCG (Multi-cloud gateway) resources that are currently in use. Those resources are used to store data according to the buckets policies and can be a cloud-based resource or a bare metal resource. 

**Health:**
The object storage health state consists of the system’s data availability, buckets, and resources health and functionality.

**Data resiliency:** 
Data resiliency is the ability of stored objects to recover and continue operating in the case of a failure. 

Certain changes in the system (unavailable resource/ change of bucket policy etc.) cause an object to require a rebuilding process in order to stay resilient. 

**Efficiency ratio:**
Efficiency ratio refers to the deduplication and compression process effectiveness. 

**Savings:**
Savings shows the uncompressed and non-deduped data that would have been stored without those techniques. 

"See All" Link should show a subsystem Health card (such as the cluster dashboard) assuming it can be implemented in 4.2 
![image](https://user-images.githubusercontent.com/10331355/60430364-cf684f00-9c05-11e9-859c-5b2ea737b3d9.png)

If not possible, we can remove the "See all" link for now

All other "view x" should link to the MCG UI in its relevant page.

Reply:
Removing it for 4.2 sounds like a good idea. @ julienlim @etamir

Reply:
If it's supported, why should we remove? (if not, of course, we'll remove)